plugins/autocertkit

```diff
@@ -1632,9 +1632,18 @@ def _get_local_device_bridge(session, device):
             log.debug("%s is a bridge." % device)
             return device
 
-    raise Exception("Error: Cannot determine bridge name of given device name %s." % device)
+    log.debug("Cannot determine bridge of given device %s." % device)
 
-    return devname
+    if device.startswith('xenbr'):
+        log.debug("Try using device name %s." % device)
+        return device
+
+    elif device.startswith('eth'):
```

Comment:
@rdobson 

Newly created network, meaning logical network like bond or vlan, which will be filtered code above, if not, as it does not start with xenbr or eth, it will raise exception.

Native interface (any ethX) should have corresponding dummy bridge regardless. (at least across all XS version until so far) If we need to check existence of dummy bridge, then it would be better checking this in preflight than every time this happens. As native device does not change through out ACK run.

What do you think?